packages/reports/addon/templates/components/report-view.hbs

```diff
@@ -1,7 +1,7 @@
 {{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <div class="report-view__visualization-main">
   <ReportViewOverlay
-    @runReport={{route-action "runReport" @report}}
+    @runReport={{pipe (route-action "validate" @report) (route-action "forceRun" @report)}}
```

Comment:
What is the motivation behind this change?

Reply:
The goal was to align with
https://github.com/yavin-dev/framework/blob/d25799becb397105f69a4b5a6d0fd07b941152e6/packages/reports/addon/templates/reports/report.hbs#L34
I noticed while testing that you could attempt to run the report from here and it wouldn't redirect to /invalid